chainservice/builder.go

```diff
@@ -559,16 +572,20 @@ func (builder *Builder) registerStakingProtocol() error {
 	if !builder.cfg.Chain.EnableStakingProtocol {
 		return nil
 	}
-
+	consensusCfg := consensusfsm.NewConsensusConfig(builder.cfg.Consensus.RollDPoS.FSM, builder.cfg.DardanellesUpgrade, builder.cfg.Genesis, builder.cfg.Consensus.RollDPoS.Delay)
```

Comment:
seems a little weird way of getting the `BlockInterval` func. Is it possible to get it a better way?

Reply:
there is similar code at https://github.com/iotexproject/iotex-core/blob/master/chainservice/builder.go#L669